### PR TITLE
Updating to version 7.1.2

### PR DESCRIPTION
**Pull request by dtchanpura**
These are done by automated script. After the change the paragraph tags were merged together.

### CODE REVIEW

**Comment by dac73** [comment]
Thanks for PR.
Merged <p> tags will be kind of ugly on flathub. Do you think we can do this some other way?

File: io.dbeaver.DBeaverCommunity.appdata.xml [hunk 1]
```diff
@@ -30,6 +30,55 @@
...
+                    <li>Icons for "Create object" actions were fixed</li>
+                    <li>Main menu File items were reordered</li>
+                    <li>Heading and trailing spaces trim was added to all edit dialogs/panels</li>
+                </ul>
+                <p>Session manager: problem with multiple windows was fixed
+                SQL Server: Database tools (index rebuild, triggers disable) were redesigned
+                Firebird: new table and table column creation was fixed
+                PostgreSQL: Table partition DDL was fixed
+                MySQL: CHECK constraints edit support was added
+                Oracle:</p>
```
Comment (dac73) [inline io.dbeaver.DBeaverCommunity.appdata.xml:61]: In this case, 'Oracle' should be separate `<p>` tag.
Update script still needs some refining in order to be automated...

Comment (dtchanpura) [reply io.dbeaver.DBeaverCommunity.appdata.xml:61]: Got it, I missed those parts.

**Comment by dtchanpura** [comment]
Can we add `<br>` to have new ones on new lines atleast?

**Comment by dac73** [comment]
`<br>` looks good on paper, let's try that.

**Comment by dac73** [comment]
bot, build

**Comment by dtchanpura** [comment]
Let me build it locally and push it if it works well on my system.

**Comment by dac73** [comment]
Nah, it will fail because of XML validation.

You can validate localy(ln 125 in update.py), but flathub has a bit different rules

**Comment by dac73** [comment]
Should we just replace release notes with a link to release notes?

For this release just remove `<br>` 
I don't know is there any other tag we can use for multiline text in AppData.

**Comment by dtchanpura** [comment]
We could certainly try that but, as per this [link](https://www.freedesktop.org/software/appstream/docs/chap-CollectionData.html#tag-ct-description) it currently only supports paragraph, ordered list and unordered list. I don't think we can add `<a href>`

**Comment by dac73** [comment]
Yeah, then we are stuck with the current version, just remove `<br>`

**Comment by dac73** [comment]
@dtchanpura can you please add interpunction in multiline paragraphs?

**Comment by dtchanpura** [comment]
Sure, should I just add `,` or `.` to them?

**Comment by dac73** [comment]
bot, build

**Comment by dtchanpura** [comment]
I was also thinking if nested structure could work. like as the change log has list of list we could try that as well. but not sure if that's supported or already tried.

**Comment by dac73** [comment]
Nested lists are not supported (I think I tried that first).

**Pull request merged**